packages/valory/agents/oracle/aea-config.yaml

```diff
@@ -138,23 +138,13 @@ public_id: valory/p2p_libp2p_client:0.1.0
 type: connection
 config:
   nodes:
-  - uri: ${P2P_URI:str:acn.staging.autonolas.tech:9005}
-    public_key: ${P2P_PUBLIC_KEY:str:02d3a830c9d6ea1ae91936951430dee11f4662f33118b02190693be835359a9d77}
-  - uri: ${P2P_URI:str:acn.staging.autonolas.tech:9006}
-    public_key: ${P2P_PUBLIC_KEY:str:02e741c62d706e1dcf6986bf37fa74b98681bc32669623ac9ee6ff72488d4f59e8}
+  - uri: localhost:11000
+    public_key: 03c74dbfbe7bbc1b42429f78778017a3cd7eaf9d59d1634c9505a3f7c1a9350e71
```

Comment:
We want to start with single node

Reply:
Why? And if so, why change the public_key and get rid of the env var?

Reply:
Continued from https://github.com/valory-xyz/open-autonomy/pull/1432#discussion_r996609138, as I said these are just place holders which can be overridden at the runtime using service level overrides. But for that first we need to implement the override policies and update the service component

Reply:
same here: `${P2P_PUBLIC_KEY:str:03c74dbfbe7bbc1b42429f78778017a3cd7eaf9d59d1634c9505a3f7c1a9350e71}

Reply:
We should have a list somewhere in the docs with the local and staging addresses